code/modules/mob/living/simple_animal/friendly/drone/interaction.dm

```diff
@@ -113,6 +113,12 @@
 	else
 		..()
 
+/mob/living/simple_animal/drone/transferItemToLoc(obj/item/item, newloc, force, silent)
+	. = ..()
+	for (var/tool in internal_storage)
+		if (istype(item, tool))
+			return FALSE
```

Comment:
I don't understand this loop.

`internal_storage` is an `/obj/item`. Can this not just be `return item != internal_storage && ..()`?

Reply:
internal storage is the thing that stores all of their tools.
![image](https://user-images.githubusercontent.com/16159590/148567831-f33a83de-7662-4ed4-927d-c412f6f5b370.png)
`return item != internal_storage` would never be true

Reply:
`in` would work though